test/built-ins/Function/prototype/arguments/prop-desc.js

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2024 Justin Dorfman. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-function.prototype.arguments
+description: >
+  Function.prototype.arguments property descriptor
+info: |
+  Function.prototype.arguments is an accessor property whose set and get
+  accessor functions are both %ThrowTypeError%.
+includes: [propertyHelper.js]
+---*/
+
+const argumentsDesc = Object.getOwnPropertyDescriptor(Function.prototype, 'arguments');
+
+verifyProperty(Function.prototype, "arguments", {
+  enumerable: false,
+  configurable: true
+});
+
+assert.sameValue(typeof argumentsDesc.get, "function",
+  "Function.prototype.arguments has function getter");
+assert.sameValue(typeof argumentsDesc.set, "function",
+  "Function.prototype.arguments has function setter");
+assert.sameValue(argumentsDesc.get, argumentsDesc.set,
+  "Arguments property getter/setter are the same function");
```

Comment:
```suggestion
  "Function.prototype.arguments property getter/setter are the same function");
```

And we also want to assert that the getter/setter is in fact %ThrowTypeError%.
```diff
-includes: [propertyHelper.js]
+includes: [propertyHelper.js, wellKnownIntrinsicObjects.js]
```
```suggestion
  "Function.prototype.arguments property getter/setter are the same function");

var throwTypeError;
WellKnownIntrinsicObjects.forEach(function(record) {
  if (record.name === "%ThrowTypeError%") {
    throwTypeError = record.value;
  }
});
if (throwTypeError) {
  assert.sameValue(descriptor.set, throwTypeError, "Function.prototype.arguments getter is %ThrowTypeError%");
}
assert.throws(TypeError, function() {
  return Function.prototype.arguments;
});
assert.throws(TypeError, function() {
  Function.prototype.arguments = {};
});  
```

_(and likewise in **test/built-ins/Function/prototype/caller/prop-desc.js**)_